packages/shared/src/modules/users/libs/validation-schemas/user-update.validation-schema.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from "zod";
+
+import { UserValidationMessage, UserValidationRule } from "../enums/enums.js";
+
+type UserUpdateRequestValidationDto = {
+	name: z.ZodString;
+};
+
+const userUpdate = z
+	.object<UserUpdateRequestValidationDto>({
+		name: z.string().trim().min(UserValidationRule.NAME_MINIMUM_LENGTH, {
```

Comment:
Should we check user name for some special characters?

Reply:
I will check requirements for user field.